feature-add-meal/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: "com.android.library"
+apply from: rootProject.file("common.gradle")
+
+def composeVersion = "1.0.0-alpha07"
+
+dependencies {
+    // Feature module
+    implementation project(":lib-theming")
```

Comment:
Nice that you are adding comments to the gradle file to help improve the readability 👍